avm.tflint.hcl

```diff
@@ -136,6 +136,62 @@ rule "azurerm_resource_tag" {
   enabled = true
 }
 
+# AVM Provider Rules
+
 rule "tfnfr26" {
   enabled = true
-}
\ No newline at end of file
+}
+
+# AVM Module Rules
+
+rule "required_module_source_tffr1" {
+  enabled = true
+}
+
+# AVM Output Rules
+
+rule "required_output_rmfr7" {
+  enabled = true
+}
+
+rule "required_output_tffr2" {
+  enabled = true
+}
+
+# AVM Variable Interface Rules
+
+rule "customer_managed_key" {
+  enabled = true
+}
+
+rule "diagnostic_settings" {
+  enabled = true
+}
+
+rule "location" {
+  enabled = true
+}
+
+rule "lock" {
+  enabled = true
+}
+
+rule "managed_identities" {
+  enabled = true
+}
+
+rule "private_endpoints" {
+  enabled = true
+}
+
+rule "private_endpoints" {
+  enabled = true
+}
+
+rule "role_assignments" {
+  enabled = true
```

Comment:
These rules are enabled by default, do we need them here?

Reply:
I was unsure about this, but I thought surfacing them would be more user friendly for those that want to override them. It helped me to understand what rules are included in avm. But also appreciate your point. I happy either way. Can remove them if you like.

Reply:
The others are in there, so we can keep them in - all good